app/views/home/index.html.haml

```diff
@@ -15,12 +15,13 @@
       %li.mr-2
         %a.inline-block.p-4.text-black.bg-emerald-300.hover:bg-emerald-400{href: "#"} Categories
       %li.ml-auto
+        %button.text-black.p-3.flex.items-center.hover:bg-emerald-300
+
         %button.text-black.p-3.flex.items-center.hover:bg-emerald-400
-          -# = link_to "plants" plants
           %svg.mr-2{fill: "#047857", height: "28", version: "1.1", viewbox: "0 0 26 28", width: "26", xmlns: "http://www.w3.org/2000/svg"}
             %title search1
             %path{d: "M18 13c0-3.859-3.141-7-7-7s-7 3.141-7 7 3.141 7 7 7 7-3.141 7-7zM26 26c0 1.094-0.906 2-2 2-0.531 0-1.047-0.219-1.406-0.594l-5.359-5.344c-1.828 1.266-4.016 1.937-6.234 1.937-6.078 0-11-4.922-11-11s4.922-11 11-11 11 4.922 11 11c0 2.219-0.672 4.406-1.937 6.234l5.359 5.359c0.359 0.359 0.578 0.875 0.578 1.406z"}
-          %p.text-base Search for plants
+          = link_to "Search for plants" , people_path, :class => "text-base"
```

Comment:
This search_box is basically a link to people_path, I think you should look at replicating the search function on the plants index page.

Reply:
No, that was not my intention to make a search function but rather a link towards the /people page. 

I know my UI choice is bizarre but I could refactor it later to make look like a real link.